db/seeds.rb

```diff
@@ -6,12 +6,58 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
-sample_subscribers = [
-  { username: "Schwarzenegger", uid: "12044602" , provider: "twitter", avatar_url: "https://pbs.twimg.com/profile_images/665340796510466048/-nsoU1Q5.jpg" }
+sample_subscriptions = [
+  { username: "Schwarzenegger", uid: "12044602" , provider: "twitter", avatar_url: "https://pbs.twimg.com/profile_images/665340796510466048/-nsoU1Q5.jpg" },
+  { username: "Robot", uid: "10333086" , provider: "vimeo", avatar_url: "https://i.vimeocdn.com/portrait/8242122_300x300.webp" },
+  { username: "schwarzenegger", uid:"198945880", provider:"instagram", avatar_url:"https://scontent-sea1-1.cdninstagram.com/hphotos-xta1/t51.2885-19/11373921_1614186788830308_1200274240_a.jpg"}
 ]
 
-sample_stories = Twitter.user_time("Schwarzenegger")
+sample_subscriptions.each do |subscription|
+  Subscription.create(subscription)
+end
+
+
+client =  Seemore::Application.config.twitter
+sample_stories = client.user_timeline("Schwarzenegger")
+sample_stories.each do |story|
+  new_story = Story.new
+  new_story.uid = story.id
+  new_story.text = story.text
+  if !story.media.empty?
```

Comment:
Putting a space between the variable assignments and the conditional would make this logic much easier to read.